lib/white_bread/suite_configuration.ex

```diff
@@ -0,0 +1,68 @@
+defmodule WhiteBread.SuiteConfiguration do
+  alias WhiteBread.Suite.DuplicateSuiteError
+
+  defmacro __using__(_opts) do
+    quote do
+      import WhiteBread.SuiteConfiguration
+
+      @suites []
+      @before_compile WhiteBread.SuiteConfiguration
+    end
+  end
+
+  defmacro __before_compile__(_env) do
+    quote do
+      def suites do
+        @suites
+          |> throw_if_not_unique
+          |> Enum.reverse
+      end
+
+      defp throw_if_not_unique(suites) do
```

Comment:
I think the elixir convention is to put the `!` at the end of a function that throws an exception. In your case you are raising the `DuplicateSuiteError`, as opposed to returning a tuple containing error info.
Could rename to: unique!